gson/src/main/java/com/google/gson/stream/JsonWriter.java

```diff
@@ -162,6 +167,190 @@
     HTML_SAFE_REPLACEMENT_CHARS['\''] = "\\u0027";
   }
 
+  private class StringValWriter extends Writer {
```

Comment:
StringValueWriter  since Google style is to not abbreviate. Perhaps there's a better name since StringWriter is taken?

Reply:
I have changed it to `StringValueReader` and `StringValueWriter` now. However, these classes are not publicly visible so their names can be changed at any point.